requirements.txt

```diff
@@ -2,7 +2,7 @@ Django==1.6.11
 Markdown==2.5
 PyJWT==0.1.2
 South==1.0
-amqp>=1.4.9
+amqp<2.0
```

Comment:
I was wondering whether it would make sense to have `>=1.4.0,<2.0` but there are no other releases in between. Good :)